README.md

```diff
@@ -79,6 +79,7 @@ containerd offers a full client package to help you integrate containerd into yo
 import (
   "github.com/containerd/containerd"
   "github.com/containerd/containerd/cio"
+  "github.com/containerd/containerd/namespaces"
```

Comment:
Maybe, we should add `import  "github.com/containerd/containerd/namespaces"` and `import context` here:

![image](https://user-images.githubusercontent.com/9248427/117002603-ba3b3a80-ad1e-11eb-9927-b44a854ef488.png)

Reply:
Thanks, forgot to add `context` here. I added at top of the imports.